Sources/FloatLabelTextField.swift

```diff
@@ -82,10 +83,26 @@ import UIKit
         }
     }
     
-    @IBInspectable public var titleActiveTextColor:UIColor! {
+    @IBInspectable public var titleActiveTextColor:UIColor? {
+        didSet {
+            overrideTintColorActiveColor = (titleActiveTextColour != nil)
+            
+            if overrideTintColorActiveColor {
+                titleCustomActiveTextColour = titleActiveTextColour
```

Comment:
You are using the old `colour` version here and above. I don't think it will compile.

Reply:
Resolved! Sorry about that — the example workspace doesn't actually compile, which makes these changes difficult to quickly iterate on.

Reply:
Haven't run the examples 😄 . Thanks for the heads-up